release/pkg/manager/calico/manager.go

```diff
@@ -157,13 +176,19 @@ func (r *ReleaseBuilder) Build() error {
 		return fmt.Errorf("failed to create output dir: %s", err)
 	}
 
-	if !r.isHashRelease {
-		if r.validate {
+	if r.validate {
```

Comment:
I think this would be nicer like this:

```go
if r.validate {
  r.Validate(ver)
}
```

```go
func (r *CalicoManager) Validate(version string) error {
  if r.isHashRelease {
    return r.HashReleaseValidate()
  }
  return r.PreReleasevalidate()
}
```

Fine for another PR, though.